third_party/Makefile

```diff
@@ -18,14 +18,22 @@ NUM_MAKE_JOBS ?= $(shell nproc --ignore=2)
 
 .PHONY: all cuda_with_msccl cuda rocm common cuda_cutlass cuda_bandwidthTest cuda_nccl_tests cuda_perftest cuda_msccl rocm_perftest fio rocm_rccl_tests rocm_rocblas rocm_bandwidthTest gpcnet cuda_gpuburn cpu_stream cpu_hpl directx_amf_encoding_latency directx_amd rocm_hipblaslt megatron_lm megatron_deepspeed apex_rocm
 
-# Build all targets.
+# Build targets.
+ifeq ($(shell uname -m), aarch64)
```

Comment:
maybe we can just put `common` and `directx_amd` targets inside ifeq and put the rest outside?

Reply:
cleaned it to remove repetition of targets